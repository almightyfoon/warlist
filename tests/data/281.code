c4501b_-0B0Z191u1ui22EdWdWhZh_hZh____-0g0x19191x2DhC2k2t2t2b23203g203g203g