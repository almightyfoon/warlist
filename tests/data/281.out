https://conflictchamber.com/#c4501b_-0B0Z191u1ui22EdWdWhZh_hZh____-0gkv19191x2DhC2k2t2t2bkz203g203g203g

Cryx Army - 75 / 75 points
[Theme] Dark Host

[Coven 1] The Witch Coven of Garlghast [+26]
 - Deathripper [6]
 - Seether [13]
 - Seether [13]
Bane Lord Tartarus [0(6)]
Darragh Wrathe [9]
Bane Riders (max) [20]
Bane Riders (max) [20]
Bane Warriors (min) [10]
 - Bane Warrior Officer & Standard [0(5)]
Bane Warriors (min) [10]
 - Bane Warrior Officer & Standard [0(5)]

Cryx Army - 78 / 75 points
[Theme] The Ghost Fleet

!!! This army must contain 75 points or fewer

[Deneghra 1] Warwitch Deneghra [+26]
 - Deathripper [6]
 - Deathripper [6]
 - Nightmare [18]
Captain Rengrave [5]
Hellslinger Phantom [0(7)]
Machine Wraith [2]
Pistol Wraith [0(5)]
Pistol Wraith [0(5)]
Blackbane's Ghost Raiders (max) [17]
Revenant Cannon Crew [5]
Revenant Crew of the Atramentous (min) [9]
 - Revenant Crew Rifleman (3) [6]
Revenant Crew of the Atramentous (min) [9]
 - Revenant Crew Rifleman (3) [6]
Revenant Crew of the Atramentous (min) [9]
 - Revenant Crew Rifleman (3) [6]
